aries-site/src/pages/components/spinner.mdx

```diff
@@ -0,0 +1,69 @@
+import { Example } from '../../layouts';
+import {
+  AnnounceSpinnerExample,
+  SpinnerExample,
+  ListSpinnerExample,
+  ContentSpinnerExample,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/spinner/SpinnerExample.js"
+  docs="https://v2.grommet.io/spinner?theme=hpe"
+  figma="https://www.figma.com/file/feIeYk1zX36y9E7e7rprc2/HPE-Loading-Indicator-Component?node-id=0%3A1"
+>
+  <SpinnerExample />
+</Example>
+
+## Guidance
+
+Loading Spinners are used to notify the user that an action is being processed. 
+This reassures the user that something is occuring and will be provided shortly. 
+
+Progress circles help the user to show the progression of a system operation (downloadnig, uploading, processing, etc) visually. 
+### When to use Spinner
+
+Use a spinner when an action requires a short time to process (downloadnig, uploading, processing, etc).
+When loading more information into a table or list a spinner should be used.
+
+### Accessibility
+
+In order for a spinner to be accessible we need to provide a [message](https://v2.grommet.io/spinner#message) that will be announced for screen readers. 
+
+## Variants
+
+A spinner is flexable to used and should be used to give verifcation to the user that something is happening upon their request. 
```

Comment:
Suggestion to replace: A spinner can be used in various ways to indicate to the user that a task is in process.